content/api/commands/origin.md

```diff
@@ -507,27 +507,37 @@ into the callback.
 
 ### Dependencies / Sharing Code
 
-It is not possible to use
+[ES module dynamic `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
+and/or
 [CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-or
-[dynamic ES module `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
-within the callback. However, [`Cypress.require()`](/api/cypress-api/require)
-can be utilized to include [npm](https://www.npmjs.com/) packages and other
-files. It is functionally the same as using
-[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-in browser-targeted code.
+can be used within the callback to include [npm](https://www.npmjs.com/)
+packages and other files.
+
+If you utilize a custom preprocessor, note that this feature requires the latest
```

Comment:
Suggest calling out the actual version, e.g. requires version 5.14.0 or greater

Reply:
I'm more inclined to do that here than in the code, since it can be changed at will if something goes wrong with the package release. It should get released once the code PR is in develop, so I'll update this with the version then.

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/4789/commits/42354f2865a0abe95011ba798769dfff2879a8df